Add wfc - wave function collapse by @krychu

crisdosaygo opened the pull request:
Just spotted this today and thought it would be a good addition @nothings 

Hope you're well!

Include the link to the library in your description, as well as in the PR itself.

crisdosaygo committed:
Add wfc - wave function collapse by @krychu

Just spotted this today and thought it would be a good addition @nothings 

Hope you're well!

krychu commented:
@i5ik thanks a lot! It'd be great to have it listed here. Would it fit better in `graphics (2d)`?

crisdosaygo commented on README.md:
```diff
@@ -243,6 +243,7 @@ as C/C++, as this is not an obstacle to most users.)
 |  [Vulkan Memory Allocator](https://github.com/GPUOpen-LibrariesAndSDKs/VulkanMemoryAllocator)|MIT|C/C++|**1**| memory allocator for Vulkan
 |  [yocto_trace.h](https://github.com/xelatihy/yocto-gl)                | MIT                  |C/C++|**1**| physically-based unidirectional path tracer w/ MIS for direct lights
 |  [yocto_symrigid.h](https://github.com/xelatihy/yocto-gl)             | MIT                  |C/C++|**1**| rigid body simulator (sequential impulse/PGS) with support for concave objects
+|  [wfc](https://github.com/krychu/wfc)                                 | MIT                  |  C  |**1**| Wave Function Collapse library using the overlapping method, plus a command-line tool
```
Hey @nothings and @krychu , re https://github.com/nothings/single_file_libs/pull/239#issuecomment-1006134597 I think it would be better in `graphics (2d)`

crisdosaygo replied:
I just moved it it in https://github.com/i5ik/single_file_libs/commit/6c7a53ac3b41147aa488f249578edf6e35467604

crisdosaygo committed:
Move wfc from graphics (3d) to graphics (2d)

Per @krychu https://github.com/nothings/single_file_libs/pull/239#issuecomment-1006134597

crisdosaygo reviewed (commented):
Hey @nothings good to see you're here! 

I'm sure you must be super busy, so I'm sorry for the following very annoying request, but if this LGTY, I'm sure @krychu and I would appreciate a merge. 

Anyway, I'm sure you're super busy so I get if you're too busy to reply or if anything else makes this impossible for you--and if so no worries at all!--just thank you for reading this far!

(and thanks ❤️ for making [all the cool things](https://github.com/nothings))

Have a good one!💪🏻😹

nothings commented:
the PR is fine, I just only update this list in batches once in a while.
